gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -192,9 +202,53 @@ protected void processMessage(DecodeableKafkaRecord message) {
     dagActionsSeenCache.put(changeIdentifier, changeIdentifier);
   }
 
-  protected void submitFlowToDagManagerHelper(DagActionStore.DagAction dagAction) {
+  /**
+   * For a given dagAction, calls the appropriate method in the DagManager to carry out the desired action.
+   * @param isStartup true if called for dagAction loaded directly from store upon startup, false otherwise
+   */
+  private void handleDagAction(DagActionStore.DagAction dagAction, boolean isStartup) {
+    log.info("(" + (isStartup ? "on-startup" : "post-startup") + ") DagAction change ({}) received for flow: {}",
+        dagAction.getFlowActionType(), dagAction);
+    if (dagAction.getFlowActionType().equals(DagActionStore.FlowActionType.RESUME)) {
```

Comment:
On the left side, this piece of code was written with switch-case, now it is changed to if-else. Why?
IntelliJ complains when if-else is used.